progressive-web-app/.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,60 @@
+name: AIO App CI
+
+on:
+  release:
+    types: [released]
+jobs:
+  deploy:
+    name: Deploy to Prod
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        node-version: ['16']
```

Comment:
might as well use the latest?

Reply:
the workflows are not really required to demonstrate how this code works, so I'll just remove them all.